docs/snippets/all/archetypes/depth_image_3d.py

```diff
@@ -20,4 +20,4 @@
 )
 
 # Log the tensor.
-rr.log("world/camera/depth", rr.DepthImage(depth_image, meter=10_000.0, colormap="viridis"))
+rr.log("world/camera/depth", rr.DepthImage(depth_image, meter=10_000.0, backproject_radius_scale=0.2))
```

Comment:
I'd keep the colormap in this example.

Reply:
oops. thanks, that was a mistake, I just wanted to try this out briefly and don't push it. would have left out the backproject radius from the example since I don't think it's all that useful to show